yt/yt/server/queue_agent/queue_controller.cpp

```diff
@@ -576,19 +589,32 @@ class TOrderedDynamicTableController
             GuardedTrim();
         } catch (const std::exception& ex) {
             YT_LOG_ERROR(ex, "Error while trimming queue");
+            TrimAlertCollector_->StageAlert(CreateAlert(
+                NAlerts::EErrorCode::QueueAgentQueueControllerTrimFailed,
+                "Error while trimming queue",
+                /*tags*/ {},
+                ex));
         }
+
+        TrimAlertCollector_->PublishAlerts();
     }
 
     struct TPartitionTrimContext
     {
         int PartitionIndex;
         TError PartitionError;
+        bool IsErrorCritical = true;
```

Comment:
Should it be with `true` default value?

Reply:
IMO yes, it is rather unusual for an error in trimming to be "ok". Non-mounted tables are pretty much the only case.